mir-module/src/main/resources/META-INF/resources/js/epusta.js

```diff
@@ -185,45 +185,45 @@ OASGraph.prototype= {
         default:
           from.setMonth(today.getMonth() - 12);
       }
-      return from.toJSON().substring(0,10);  
+      return from.toJSON().substring(0,10);
     } else {
       return this.from;
     }
-  } 
+  }
 };
 
-OASGraph.receiveData = function(oasgraph,json) {
+ePuStaGraph.receiveData = function(epustagraph,json) {
   if (json) {
-    oasgraph.data=json.entries;
-    oasgraph.state="success";
+    epustagraph.data=json.entries;
+    epustagraph.state="success";
   } else {
-    oasgraph.state="error";
-    oasgraph.errortext="Keine Zugriffsdaten vorhanden";
+    epustagraph.state="error";
+    epustagraph.errortext="Keine Zugriffsdaten vorhanden";
   }
-  oasgraph.render();
+  epustagraph.render();
 };
 
-// End Class OASGraph
+// End Class ePuStaGraph
 
 $(document).ready(function() {
-  $('[data-oaselementtype]').each(function(index, element) {
-    var oasElementtype=$(element).data('oaselementtype');
-    var oasProviderurl=$(element).data('oasproviderurl');
-    var oasIdentifier=$(element).data('oasidentifier');
-    var oasCounttype=$(element).data('oascounttype');
-    var oasFrom=$(element).data('oasfrom');
-    var oasUntil=$(element).data('oasuntil');
-    var oasElement;
-    if (oasElementtype == "OASInline" ) {
-      oasElement = new OASInline(element,oasProviderurl,oasIdentifier,oasFrom,oasUntil,oasCounttype);
-      oasElement.requestData();
+  $('[data-epustaelementtype]').each(function(index, element) {
+    var epustaElementtype=$(element).data('epustaelementtype');
+    var epustaProviderurl=$(element).data('epustaproviderurl');
+    var epustaIdentifier=$(element).data('epustaidentifier');
+    var epustaCounttype=$(element).data('epustacounttype');
+    var epustaFrom=$(element).data('epustafrom');
+    var epustaUntil=$(element).data('epustauntil');
+    var epustaElement;
+    if (epustaElementtype == "ePuStaInline" ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5639065)